Kumi.Game/KumiGame.cs

```diff
@@ -89,6 +93,16 @@ protected override void LoadComplete()
 
         Task.Factory.StartNew(() =>
         {
+            // This has to be manually added first because of some weird fuckery i can't explain at the moment
+            LoadComponent(ControlOverlay = new ControlOverlay
+            {
+                Anchor = Anchor.TopRight,
+                Origin = Anchor.TopRight
+            });
+            
+            DependencyContainer.CacheAs<INotificationManager>(ControlOverlay);
```

Comment:
What

Reply:
don't question it lmfao, i'll re-write the loader code to something better.